runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -93,13 +88,10 @@ function M.on_inlayhint(err, result, ctx, _)
 end
 
 --- |lsp-handler| for the method `workspace/inlayHint/refresh`
----@param ctx lsp.HandlerContext
+---@param client_id integer
 ---@private
-function M.on_refresh(err, _, ctx, _)
-  if err then
-    return vim.NIL
-  end
-  for _, bufnr in ipairs(vim.lsp.get_buffers_by_client_id(ctx.client_id)) do
+function M.on_refresh(client_id)
+  for _, bufnr in ipairs(vim.lsp.get_buffers_by_client_id(client_id)) do
```

Comment:
Are you sure this is correct? `workspace/inlayHint/refresh` is for a request that's sent from the server to the client instead of the usual client->server handler.

These do have a response that need to be sent to the server. Same for semantic tokens below

Reply:
Sorry this is wrong, I was meant to revert this.